src/main/kotlin/no/nav/familie/ef/sak/økonomi/TilkjentYtelseController.kt

```diff
@@ -23,7 +21,7 @@ class TilkjentYtelseController(private val tilkjentYtelseService: TilkjentYtelse
         val tilkjentYtelseId = tilkjentYtelseService.opprettTilkjentYtelse(tilkjentYtelseDTO)
 
         val location = ServletUriComponentsBuilder.fromCurrentRequestUri()
-                .path(tilkjentYtelseId.toString())
+                .path("/" + tilkjentYtelseId.toString())
```

Comment:
Litt rart att den ikke legger på "/" selv. Kotlin støtter string template, `"/$tilkjentYtelseId"`

Reply:
Hvis du bruker pathSegment i stedet for path, så legger den til / for deg.